frontend/src/metabase/entities/pulses.js

```diff
@@ -31,37 +64,14 @@ const Pulses = createEntity({
       );
     },
 
-    setChannels: ({ id }, channels, opts) => {
-      return Pulses.actions.update(
-        { id },
-        { channels },
-        undo(opts, t`subscription`, t`updated`),
-      );
-    },
-
-    setCollection: ({ id }, collection, opts) => {
-      return Pulses.actions.update(
-        { id },
-        { collection_id: canonicalCollectionId(collection && collection.id) },
-        undo(opts, t`subscription`, t`moved`),
-      );
-    },
-
-    setPinned: ({ id }, pinned, opts) => {
-      return Pulses.actions.update(
-        { id },
-        {
-          collection_position:
-            typeof pinned === "number" ? pinned : pinned ? 1 : null,
-        },
-        opts,
-      );
-    },
-
     unsubscribe:
       ({ id }) =>
       async dispatch => {
```

Comment:
I think you can remove the `async` here.

```suggestion
     dispatch => {
```

Reply:
That was here from before so I just went with it.
It's safer to leave it because we'll remove the whole thing in the milestone 2, anyway.